index.js

```diff
@@ -73,9 +73,16 @@ for (let i = 0; i < answerButtons.length; i++) {
   })
 }
 
-//Toogle Bookmarks
+// Toogle Bookmarks
 for (let i = 0; i < bookmarks.length; i++) {
   bookmarks[i].addEventListener('click', () => {
     bookmarks[i].classList.toggle('card__bookmark--inactive')
   })
 }
+// Reset Form
+form = document.getElementById('form')
+submitButton = getEl('.input__button')
+submitButton.addEventListener('click', () => {
+  form.reset()
+  createButton.click()
+})
```

Comment:
This does not work, right? 
`submitButton` and `form` need to be _saved_ as a variable. And for the form we have a different kind of event we need to trigger. Have a look at this [codepen](https://codepen.io/macfrei/pen/mdwOEBd?editors=0010).
Also can recommend this article from mdn: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event